versions.yaml

```diff
@@ -65,5 +68,24 @@ versions:
           gpg: *gpg
           version: '1.14.2'
 
+  - dir: 'exporter'
+    repo: nginx-prometheus-exporter0
+    templateSubDir: exporter
+    templateArgs:
+      repo: 'http://nginx.org/packages/mainline/debian/'
+      os: 'stretch'
+    tags:
+      - '0.3.0-debian9'
```

Comment:
Please follow the convention. It's good to always take a look at different repos.

Reply:
Done. Using consul-docker as a template.